Sources/ALDesign/Component/Tag/ALTag.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ALTag.swift
+//  
+//
+//  Created by 이옥민 on 2022/04/18.
+//
+
+import UIKit
+
+public class ALTag: UIView {
+    lazy private var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private let type: ALTagType
+    private let backgroundColors: [UIColor]?
+    
+    public init(title: String, backgroundColor: UIColor, type: ALTagType) {
+        self.type = type
+        self.backgroundColors = nil
+        super.init(frame: .zero)
+        self.backgroundColor = backgroundColor
+        initView(title: title)
+    }
+    
+    public init(title: String, backgroundColor: [UIColor], type: ALTagType) {
```

Comment:
만들어주신 property명과 동일하게 array로 backgroundColor -> backgroundColors로 표기되면 더 좋지 않을까 합니다 :)
```suggestion
    public init(title: String, backgroundColors: [UIColor], type: ALTagType) {
```